src/psbt.js

```diff
@@ -163,6 +174,17 @@ class Psbt {
     return this;
   }
   addOutput(outputData) {
+    if (
+      arguments.length > 1 ||
+      !outputData ||
```

Comment:
Why truthy check here but undefined check below?

Reply:
Nearly everything in JavaScript is an object, an object merely needs to be checked if it exists, so truthy check is fine.